src/main/res/values/strings.xml

```diff
@@ -45,23 +45,23 @@
   <string name="downloading_format">Downloading %s</string>
   <string name="edit_repository">Edit repository</string>
   <string name="empty_search_summary">No matching applications found</string>
-  <string name="error_downloading_format">Error downloading %s</string>
-  <string name="error_syncing_format">Error syncing %s</string>
-  <string name="error_validating_format">Error validating %s</string>
+  <string name="error_downloading_format">Could not download \"%s\"</string>
+  <string name="error_syncing_format">Could not sync \"%s\</string>
+  <string name="error_validating_format">Could not validate \"%s\</string>
   <string name="fingerprint">Fingerprint</string>
-  <string name="finished_downloading_format">Finished downloading %s</string>
+  <string name="finished_downloading_format">%s downloaded</string>
   <string name="http_error_description">Invalid server response.</string>
   <string name="http_proxy">HTTP proxy</string>
-  <string name="ignore_all_updates">Ignore all updates</string>
-  <string name="ignore_this_update">Ignore this update</string>
+  <string name="ignore_all_updates">Ignore all new versions</string>
+  <string name="ignore_this_update">Ignore this new version</string>
   <string name="incompatible_features_description">Missing features:</string>
-  <string name="incompatible_older_description">This version is older than installed on your device.
-    You\'ll have to uninstall the application to proceed.</string>
+  <string name="incompatible_older_description">Older than the one installed on your device.
+    You\'ll have to uninstall the application to install it.</string>
   <string name="incompatible_platforms_description_format">Your %1$s platform is not supported.
     Supported platforms: %2$s.</string>
   <string name="incompatible_sdk_description_format">Your %1$s (SDK version %2$d) is not supported. %3$s</string>
-  <string name="incompatible_sdk_max_description_format">Maximum SDK version is %d.</string>
-  <string name="incompatible_sdk_min_description_format">Minimum SDK version is %d.</string>
+  <string name="incompatible_sdk_max_description_format">You can not have an SDK version higher than %d.</string>
+  <string name="incompatible_sdk_min_description_format">You can not have an SDK version lower than %d.</string>
```

Comment:
"You cannot have SDK version" doesn't make sense to me. I think it's better to leave it as it was.

Reply:
There is an "an", if that helps.

Reply:
Even with "an". "Minimum SDK version is 24" means that application supports only Android 7.0 and higher, which SDK version is 24. This is why "You cannot have an SDK version" doesn't make any sense.

The full dialog message after formatting will be something like "Your Android 5.0 (SDK version 21) is not supported. Minimum SDK version is 24."